tests/integration-test.sh

```diff
@@ -84,7 +84,14 @@ echo "default_project_id = $GS_PROJECT_ID" >> $botofile
 
 
 if [ -f "$outfile" ]; then
-  gsutil cp $outfile gs://vic-ci-logs
+  echo `ls -al $outfile`
```

Comment:
I'm not clear how this is different to (`echo` seems irrelevant):
```
ls -al $outfile
```

Reply:
#8086